internal/domain/node.go

```diff
@@ -53,11 +66,24 @@ func (n *CustomNode) AddAttr(key, value string) {
 	case "id":
 		n.Attrs = append(n.Attrs, Attr{key: "ID", value: value})
 		return
+	case "tabindex":
+		n.Attrs = append(n.Attrs, Attr{key: "TabIndex", value: value})
+		return
 	case "g.Text":
 		n.Attrs = append(n.Attrs, Attr{key: key, value: value})
 		return
 	default:
-		n.Attrs = append(n.Attrs, Attr{key: cases.Title(language.English).String(key), value: value})
+		if strings.ContainsRune(key, '-') {
```

Comment:
Could you tell me why you wound need a hyphanted rule?

Isn't this case solve via custom attributes?

If you want to solve Data() element I would not generalise it to hypehented then.
You would break Aria attributes, which also will need similar kind of handling because it has got Aria()

Reply:
Nvm. I read your code incorrectly. It is ok., I would just put this outside default swtich case